extensions/query-store/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "query-store",
+  "displayName": "%queryStore.displayName%",
+  "description": "%queryStore.description%",
+  "version": "0.1.0",
+  "publisher": "Microsoft",
+  "preview": false,
+  "engines": {
+    "vscode": "*",
+    "azdata": ">=1.45.0"
+  },
+  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt",
+  "icon": "images/extension.png",
+  "aiKey": "29a207bb14f84905966a8f22524cb730-25407f35-11b6-4d4e-8114-ab9e843cb52f-7380",
+  "activationEvents": [
+    "*"
+  ],
+  "main": "./out/extension",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/azuredatastudio.git"
+  },
+  "capabilities": {
+    "virtualWorkspaces": false,
+    "untrustedWorkspaces": {
+      "supported": true
+    }
+  },
+  "extensionDependencies": [
+    "Microsoft.mssql"
+  ],
+  "contributes": {
+    "commands": [
+    ],
+    "menus": {
+      "objectExplorer/item/context": [
+      ],
+      "dataExplorer/context": [
+      ],
+      "commandPalette": [
+      ]
+    }
+  },
+  "dependencies": {
+    "@microsoft/ads-extension-telemetry": "^3.0.1",
+    "vscode-nls": "^4.0.0"
+  },
+  "devDependencies": {
+    "@microsoft/azdata-test": "^3.0.1",
+    "@microsoft/vscodetestcover": "^1.2.1",
+    "@types/htmlparser2": "^3.10.1",
+    "@types/mocha": "^7.0.2",
```

Comment:
Remove any packages you aren't actively using - that way we're sure to only have the ones we actually need (easy enough to add later when you add telemetry/tests/etc)